### PR TITLE
Add components, reducers and actions to display books, form and delete books

### PR DESCRIPTION
**Pull request by adejam**
The PR does the following:

- Create `containers`, `components`, `reducers`, and `actions` directories with necessary files.
- Combine the reducers.
- Display lists of books and a dummy form intended to be used to creating a book.
- Define the Redux actions and reducers
- Dynamically create an `id` for books using `Math.random()` method.
- [live demo](https://deploy-preview-2--booklyb.netlify.app/)

### CODE REVIEW

File: src/components/BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <select className="formControl" id="category">
+        <option value="">Choose Category</option>
+        <option value="Action">Action</option>
+        <option value="Biography">Biography</option>
+        <option value="History">History</option>
+        <option value="Horror">Horror</option>
+        <option value="Kids">Kids</option>
+        <option value="Learning">Learning</option>
+        <option value="Sci-Fi">Sci-Fi</option>
+      </select>
```
Comment (bolah2009) [inline src/components/BooksForm.js:18]: I would recommend not hardcoding this values

File: src/css/reset.css [hunk 2]
```diff
@@ -1 +1,41 @@
...
+    line-height: inherit;
+}
+input:focus, select:focus, .btn:focus {
+    border-color: #00ffff;
+  }
+
+  .btn:hover {
+    opacity: 0.9;
+  }
+
```
Comment (bolah2009) [inline src/css/reset.css:41]: Can you format this file? This is ignored by styelint but not properly formatted.

File: src/index.js [hunk 3]
```diff
@@ -1,11 +1,25 @@
...
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import App from './components/App';
+import rootReducer from './reducers/index';
 
+const initialState = {
+  books: [
+    { id: Math.random(), title: 'Oliver Twist', category: 'Fiction' },
+    { id: Math.random(), title: 'Peter Pan', category: 'novel' },
+    { id: Math.random(), title: 'Superman', category: 'Sci-fi' },
```
Comment (bolah2009) [inline src/index.js:13]: This is not a good way to initialise your state, the id changes on every refresh which name not be effificent.

**Review by bolah2009 — APPROVED** [review]
Good job with the project.

**Comment by adejam** [comment]
Thanks for the review. I'll make adjustments where necessary.

File: src/components/BooksForm.js (hunk 1 above)
Comment (adejam) [reply src/components/BooksForm.js:18]: This would mean creating a component for `option` and passing a prop to it.

File: src/index.js (hunk 3 above)
Comment (adejam) [reply src/index.js:13]: I had to do it this way as using `Math.random()` as value for `id` for this milestone is part of the project requirement.

**Pull request merged**